crates/rpc/src/handlers/light_client.rs

```diff
@@ -161,3 +161,133 @@ pub async fn get_light_client_updates(
     }
     Ok(HttpResponse::Ok().json(DataVersionedResponse::new(updates)))
 }
+
+#[get("/beacon/light_client/finality_update")]
+pub async fn get_light_client_finality_update(
+    db: Data<ReamDB>,
+    req: HttpRequest,
+) -> Result<impl Responder, ApiError> {
+    // Get the latest finalized checkpoint
+    let finalized_checkpoint = db
+        .finalized_checkpoint_provider()
+        .get()
+        .map_err(|err| {
+            ApiError::InternalError(format!(
+                "Failed to get finalized checkpoint, error: {err:?}"
+            ))
+        })?
+        .ok_or_else(|| {
+            ApiError::NotFound("LC finality update unavailable".into())
```

Comment:
```suggestion
            ApiError::NotFound("Light client finality update unavailable".into())
```

could you remove the shorthands from the rest of the PR